src/lib/api.js

```diff
@@ -81,6 +81,9 @@ export const apiAuthenticated = async (path, query = null) => {
 
   if (query !== null) {
     config.params = query;
+    config.params.limit = -1;
```

Comment:
please revert this, it breaks the API for some uses (SQL error on main page).
![image](https://user-images.githubusercontent.com/2587503/105575836-78d1e100-5d66-11eb-9da3-b0064dda0e80.png)

Reply:
done